src/Fetch_Command.php

```diff
@@ -212,15 +226,15 @@ public function __invoke( $args, $assoc_args ) {
 	/**
 	 * Creates an XML string from a given array.
 	 *
-	 * Same as `\_oembed_create_xml()` in "wp-includes\embed.php" introduced in WP 4.4.0. Polyfilled as marked private (and also to cater for older WP versions).
+	 * Same as `\oembed_create_xml()` in "wp-includes\embed.php" introduced in WP 4.4.0. Polyfilled as marked private (and also to cater for older WP versions).
```

Comment:
The name of the WP Core function hasn't changed, so I think this comment needs to stay as it was originally.

Reply:
Sloppy search/replace on my part lol. Will fix tomorrow :)

Reply:
Happens to the best of us.